package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mechanical-wombat",
-  "version": "0.17.2",
+  "version": "0.17.3",
```

Comment:
If this is going in after branding, the version needs to be above 1.0.1

Reply:
I'm just going to hold it for now then, once things have calmed down i'll update it and do it then.

Reply:
right, done it now.